sycl/include/CL/sycl.hpp

```diff
@@ -11,7 +11,7 @@
 #include <CL/sycl/accessor.hpp>
 #include <CL/sycl/atomic.hpp>
 #include <CL/sycl/backend.hpp>
-#include <CL/sycl/backend/opencl.hpp>
+//#include <CL/sycl/backend/opencl.hpp>
```

Comment:
If this include isn't neccessary then why not remove it entirely? Would prefer not to introduce commented-out code.

Reply:
sure, I will remove it if no objection to make OpenCL backend API optional